src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -103,10 +106,21 @@ def update_progress(cur_loss, iteration):
                  epochs=num_epochs,
                  iterations=None)
 
+    ctx = _mxnet_utils.get_mxnet_context(max_devices=batch_size)
     model = _Model(num_classes = len(classes))
+    model_params = model.collect_params()
+    model_params.initialize(_mx.init.Xavier(), ctx=ctx)
+    if use_pretrained_model:
+        pretrained_model_url = _os.environ["PRETRAINED_MODEL_URL"]
```

Comment:
If we're going to control whether this is enabled via `use_pretrained_model`, why not just make the model URL an argument to `create`? If it's `None`, then you use random initialization